io-onboarding-pa-api/values.yaml

```diff
@@ -51,6 +51,7 @@ ioOnboardingPaApi:
     email_smtp_port: "465"
     email_smtp_secure: "true"
     email_sender: "portaleonboarding@gmail.com"
+    indicepa_administrations_url: "https://raw.githubusercontent.com/teamdigitale/io-onboarding-pa-api/master/development-data/administrations.txt"
```

Comment:
is this going to change sometimes ?

Reply:
This is the URL to the file containing the data of public administrations to be used in development environment, and its value refers to the downloadable link from the `master` branch of [io-onboarding-pa-api](https://github.com/teamdigitale/io-onboarding-pa-api) repository on github. Unless this file is moved somewhere else or the repository name changes, I can't see any reason why this value should change.